examples/platforms/cc2538/radio.c

```diff
@@ -522,18 +522,7 @@ void cc2538RadioProcess(otInstance *aInstance)
         {
             sState = OT_RADIO_STATE_RECEIVE;
 
```

Comment:
During diagnostic, the packet is sent and received under promiscuous mode, so there is no ACK request packet and no waiting ACK logic, so the code here is unnecessary.